packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -4690,6 +4693,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       onCreateChart !== undefined && model.isChartBuilderAvailable,
       model.isCustomColumnsAvailable,
       model.isFormatColumnsAvailable,
+      model.isOrganizeColumnsAvailable,
```

Comment:
I'd suggest having the `model.isOrganizeColumnsAvailable` in a separate PR - it's orthogonal to the pivot table work, and that gives us more flexibility if we need to pick it anywhere.

Reply:
Opened #2405